programs/ambassadors/ambassadors.json

```diff
@@ -38,6 +38,65 @@
           "link": "https://landscape.json-schema.org/"
         }
       ]
+    },
+    {
+      "name": "Ege Korkan",
+      "img": "https://media.licdn.com/dms/image/D4D03AQG03-U--zDJpw/profile-displayphoto-shrink_200_200/0/1707341764311?e=1724889600&v=beta&t=xY0i-IncaI51wlZVJD7fEuSb0LMdR7j0-XlipV7EK6c",
+      "bio": "Research and standardization focused engineer at Siemens with a passion to drive interoperability across all levels.",
+      "title": "Web of Things Expert at Siemens",
+      "github": "egekorkan",
+      "twitter": "egekorkan",
+      "linkedin": "ege-korkan",
+      "mastodon":"https://mastodon.social/@egekorkan",
+      "company": "Siemens AG",
+      "country": "Germany",
+      "contributions": [
+        {
+          "type": "article",
+          "title": "How the W3C Web of Things brings JSON Schema to the Internet of Things",
+          "date": {
+            "year": 2022,
+            "month": "October"
+          },
+          "link": "https://json-schema.org/blog/posts/w3c-wot-case-study"
+        },
```

Comment:
@jviotti I hope it is fine to list it here. You are the author but me and others in the W3C WoT have provided input

Reply:
For sure. Go ahead :)